### DISCUSSION TITLE
Support .WEBP file format in .EPUB

### DISCUSSION THREAD

**Discussion by stepger** [sumatrapdfreader/sumatrapdf#3414] [Ideas]
I did, using script:

 - convert all images/pictures
 - edited mime/extentions
 - edited paths
 
Pictures shows only when extention of packed book is ".**CBZ**".
When ".**EPUB**" i see placeholder "**IMAGE**"

Thank you.

**Comment by kjk** [comment]
Tracked by https://github.com/sumatrapdfreader/sumatrapdf/issues/3415
